packages/ckeditor5-show-blocks/README.md

```diff
@@ -0,0 +1,21 @@
+CKEditor 5 show blocks feature
+================================
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-show-blocks.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-show-blocks)
+[![Coverage Status](https://coveralls.io/repos/github/ckeditor/ckeditor5/badge.svg?branch=master)](https://coveralls.io/github/ckeditor/ckeditor5?branch=master)
+[![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5)
+![Dependency Status](https://img.shields.io/librariesio/release/npm/@ckeditor/ckeditor5-show-blocks)
+
+This package contains the show blocks feature for CKEditor 5. It allows you to display the HTML element names for content blocks.
+
+## Demo
+
+Check out the [demo in the show blocks feature](https://ckeditor.com/docs/ckeditor5/latest/features/show-blocks.html#demo) guide.
+
+## Documentation
+
+See the <!-- TODO - not sure if there will be an API docs page at all [`@ckeditor/ckeditor5-show-blocks` package](https://ckeditor.com/docs/ckeditor5/latest/api/show-blocks.html) --> page in [CKEditor 5 documentation](https://ckeditor.com/docs/ckeditor5/latest/) as well as the [Show blocks](https://ckeditor.com/docs/ckeditor5/latest/features/show-blocks.html) feature guide.
```

Comment:
If [select all has one](https://ckeditor.com/docs/ckeditor5/latest/api/select-all.html), this package should not be any different.